src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -265,17 +276,18 @@ private void disableSimulationItems()
          updateSimulationItemsAccordingToSimulationState();
       }
 
-      private void updateSimulationItemsAccordingToSimulationState()
+      private void updateSimulationItemsAccordingToSimulationState(bool? isRunning = null)
```

Comment:
Why is the argument defined as nullable and not just `bool`?
It seems to me that when the function is called without arguments, calling it with `_simulationState.IsRunning` would have exactly the same outcome?
Same applies to `private void updateSimulationItemsFor(Simulation simulation, bool? isRunning = null)`

Reply:
If so, I would change the argument from `bool?` to `bool `and adjust the calls.